install/minikube-installation-guide.md

```diff
@@ -0,0 +1,226 @@
+nested virtualization. If you are using a Linux system utilising Intel `VT-x` 
+and the `kvm_intel` driver, you can perform the check by:
+
+```bash
+$ cat /sys/module/kvm_intel/parameters/nested
+Y
+```
+
+If your system does not report `Y` from the `nested` parameter, then details on how
+to enable nested virtualisation can be found on the
+[KVM Nested Guests page](https://www.linux-kvm.org/page/Nested_Guests)
+
+## Check `Minikube` is running
+
+Before you install `Kata Containers`, perform a quick check that your `Minikube` is
+operating. On your guest:
+
+
+```bash
+$ kubectl get nodes
+NAME       STATUS   ROLES    AGE   VERSION
+minikube   Ready    master   78m   v1.13.4
+```
+
+Check you have virtualization enabled inside your `Minikube`:
+
+```bash
+minikube ssh
+
+            _         _ ( )           ( )           
+  ___ ___  (_)  ___  (_)| |/')  _   _ | |_      __  
+/' _ ` _ `\| |/' _ `\| || , <  ( ) ( )| '_`\  /'__`\
+| ( ) ( ) || || ( ) || || |\`\ | (_) || |_) )(  ___/
+(_) (_) (_)(_)(_) (_)(_)(_) (_)`\___/'(_,__/'`\____)
+
+# No color option, as minikube is running busybox
+$ egrep 'vmx|svm' /proc/cpuinfo
+# if you get a long line of output here, you have it enabled!
+
+$ exit
+```
+
+## Installing `Kata Containers`
+
+Now you can install the `Kata` runtime components. You will need a local copy of some `Kata`
+components to help with this, and then use the host `kubectl` (that minikube has already
+configured for you) to deploy them:
+
+```bash
+$ git clone https://github.com/kata-containers/packaging.git
+$ cd packaging/kata-deploy
+$ kubectl apply -f kata-rbac.yaml
+$ kubectl apply -f kata-deploy.yaml
+```
+
+This should have installed the `Kata` components into `/opt/kata` inside the `Minikube` node. Let's check:
+
+```bash
+$ minikube ssh
+$ cd /opt/kata
+$ ls bin
+containerd-shim-kata-v2  kata-collect-data.sh  kata-qemu     qemu-ga	     qemu-system-x86_64
+firecracker		 kata-fc	       kata-runtime  qemu-pr-helper  virtfs-proxy-helper
+$ exit
+```
+
+This shows the `Kata` components, including a `qemu`, and the `kata-runtime` for instance.
+
+## Enabling `Kata`
+
+> **Note:** This step is only required for `Minikube`/`k8s` versions <= 1.13. Since version
+> v1.14, the `RuntimeClass` is enabled by default. Performing this step on k8s > v1.14 is
+> however benign.
+
+Now the `Kata` components are installed in the `Minikube` node, you need to configure `k8s`
+`RuntimeClass` to know when to use `Kata` to run a pod.
+
+```bash
+$ curl https://raw.githubusercontent.com/kubernetes/node-api/master/manifests/runtimeclass_crd.yaml > runtimeclass_crd.yaml
+$ kubectl apply -f runtimeclass_crd.yaml
+```
+
+### Register the runtime
+
+Now register the `kata qemu` runtime with that class:
+
+```bash
+$ git clone https://github.com/clearlinux/cloud-native-setup.git
+$ cd cloud-native-setup/clr-k8s-examples
+$ kubectl apply -f 8-kata/kata-qemu-runtimeClass.yaml
+```
```

Comment:
These steps can be removed. I think we forgot to add qemu/fc runtimeclass yamls in packaging repo. @amshinde has a PR up. With that you can do everything from single packaging repo

Reply:
Done. will be updated in a bit when I refresh.